extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -132,7 +132,13 @@ export abstract class DacFxConfigPage extends BasePage {
 			return false;
 		}
 
-		let values = await this.getDatabaseValues();
+		let values: string[];
+		try {
+			values = await this.getDatabaseValues();
+		} catch (e) {
+			// if the user doesn't have access to master, just set the database to the one the current connection is to
```

Comment:
Let's add a console warn here so users can see what the error was if they want to investigate further.